sdk/mobile-center-analytics/src/main/java/com/microsoft/azure/mobile/analytics/Analytics.java

```diff
@@ -265,6 +281,43 @@ public synchronized void setInstanceEnabled(boolean enabled) {
         applyEnabledState(enabled);
     }
 
+    @Override
+    protected Channel.GroupListener getChannelListener() {
+        return new Channel.GroupListener() {
+            private static final int BEFORE_SENDING = 0;
+            private static final int SENDING_SUCCEEDED = 1;
+            private static final int SENDING_FAILED = 2;
+
+            private void callback(int type, Log log, Exception e) {
```

Comment:
You don't need to follow this logic in Crashes. Simply call callbacks from each method and remove `callback` method.
The reason of doing this in Crashes is that there are duplicated code blocks in the listener and I wanted to eliminate that.